Assets/_Project/Code/Models/Utils.cs

```diff
@@ -26,6 +26,23 @@ public static List<Y> NetworkSerializeList<T, Y>(BufferSerializer<T> serializer,
         return null;
     }
 
+    //Bidding management
+    public static void awardBiddingWinners(Team team, String categoryName)
+    {
+        if (categoryName.Equals("Czarna Skrzynka"))
+        {
+            team.BlackBoxes++;
+        }
+        else if (categoryName.Equals("Podpowiedz"))
+        {
+            team.Clues++;
+        }
+        else //wylosowano kategorie pytaniowa
+        {
+            //todo tutaj wywolac metode rozpoczynajaca etap pytania
+        }
+    }
+
```

Comment:
To powinno być w jakimś kontrolerze, np. w kontrolerze losowania, który już istnieje

Reply:
@ThePhaseless w categoryDrawManager?

Reply:
Przeniesione